lib/DDG/Spice/BikeSharing/IndegoPHL.pm

```diff
@@ -0,0 +1,33 @@
+package DDG::Spice::BikeSharing::IndegoPHL;
+use strict;
+use DDG::Spice;
+
+name 'PHL Indego Bikeshare Locations';
+source 'Indego Philadelphia';
+icon_url 'http://www.rideindego.com/';
+description 'Search Indego Bike stations in Philadelphia';
+primary_example_queries 'indego philadelphia', 'bikeshare philadelphia';
+secondary_example_queries 'philly bikeshare';
+category 'geography';
+topics 'everyday', 'travel', 'entertainment', 'geography';
+code_url 'https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/BikeSharing/IndegoPHL.pm';
+attribution github => 'marianosimone',
+            github => 'AcriCAA', 
+            web => ['http://www.marianosimone.com',  'Mariano Simone'],
+            web  => ['http://www.coreyacri.com',  'Corey Acri'],
+            web => ['https://codeforphilly.org/', "Built at Code for Philly"];
+
+triggers any => 'indego', 'bikeshare', 'bike share', 'ride indego', 'share bike';
+
+spice to => 'https://api.phila.gov/bike-share-stations/v1';
+spice wrap_jsonp_callback => 1;
+spice is_cached => 0;
+
+handle remainder_lc => sub {
+    $loc->city.' '.$_ =~ /(philadelphia|phl|philly)/i;
```

Comment:
@AcriCAA you'll need to update your tests. The IA works correctly with this.

>will this geolocate the user on mobile or through the browser? We should also add that, no? So they can find the bike stations relevant to their location. 

The `$loc` refers to the user's location using GeoIP, so it will work on mobile as well.